User-Help/FormattingHelp.md

```diff
@@ -0,0 +1,107 @@
+
+  To mark several lines as code, you can use a code fence. This is three backticks on a newline both on top and at the bottom of your lines of code. Don't use this for quotes - see below for guidance on quote formatting.
+
+* **Links**
+
+  You can generate a link using one of two methods. One way is to encase your link text in square brackets (`[ ]`) and then the target URL in parentheses (`( )`). `[Codidact](https://codidact.org)` will render as [Codidact](https://codidact.org).
+
+  You can also format a link with the inline text being `[Codidact][1]`, and then at the bottom of the post include the URL with the format `[1]: https://codidact.org`. This will look something like this:
+
+```
+
+  You can check out [Codidact][1] for an open-source, community-oriented Q&A platform!
+  
+  [1]: https://codidact.org`
+
+```
+  Which will render like this:
+
+  > You can check out [Codidact][1] for an open-source, community-oriented Q&A platform!
+  > 
+  > [1]: https://codidact.org
+
+  If you'd like to include rollover text for your link, you can put your text, surrounded by quote marks (`"`), into the URL of your link. For instance, `[Codidact](https://codidact.org "The open-source Q&A platform")` will create a link with `The open-source Q&A platform` as the rollover text.
+
+* **Blockquotes**
+
+  When quoting text from somewhere else, you can use a blockquote to signify that this is not your original text, but a quote from somewhere else. Put a greater than symbol (`>`), followed be a space, before a paragraph to mark that quote as a blockquote. For instance, `> This is a blockquote.` will render as a quote:
+
+  > This is a blockquote!
+
+  If you want to include a second paragraph as part of the same quote, you have to include another `>` before each line, like this:
+
+```
+  > This is a blockquote!
+  >
+  >This is part of the same quote!
+```
+
+  This will render as a single blockquote:
+
+  > This is a blockquote!
+  >
+  > This is part of the same quote!
+
+  See also the [citation and referencing help](/help/referencing) for guidelines on how to properly reference different quotes from various places.
+
+* **Images**
+
+  Embedding images in a post is very similar to the formatting for links (detailed above). The format `![alt text](image URL)` will embed an image in your post. Please remember to write alt text for your image! Codidact allows you to upload an image directly, or you can embed an external image from image hosting sites such as Imgur.
+
+* **Lists**
+
+  You can generate either an ordered list or a bullet list. For a bullet list, you can use either `*` or `-` to create your list.
+
+```
+* List item one
+* List item two
+```
+
+  This will render like this:
+
+>  * List item one
+>  * List item two
+
+  If you want to create a numbered list, use `1.`, `2.`, etc instead of `*` or `-`.
```

Comment:
Does Markdown automatically number things?  Like, if I have 1., 2., 3., and then I want to add something after 2, do I need to number 3 or can I just have 1., 2., 2., 3. and it'll come out 1 2 3 4?  I seem to recall that's the case (please check), and if so it'd be good to tell people they don't have to renumber 17 things by hand because they missed something at the beginning.

Reply:
Your recollection is correct - you can number everything 1 and it'll come out correctly. The corollary here is that if you don't format the list correctly (esp. if you're using complex formatting), you get everything numbered 1.